tests/random_test.py

```diff
@@ -37,6 +37,8 @@
 config.parse_flags_with_absl()
 FLAGS = config.FLAGS
 
+zip = safe_zip
```

Comment:
This looks like it isn't used in your test, but globally shadowing zip/map like this could inadvertently break other tests.

Reply:
Good point, now removed.